moped-editor/src/components/GridTable/SearchBar.js

```diff
@@ -165,7 +166,7 @@ const SearchBar = ({
         variant="outlined"
         value={searchFieldValue}
       />
-      {filterStateActive && !advancedSearchAnchor && (
+      {(filterStateActive || searchTerm) && !advancedSearchAnchor && (
```

Comment:
this works fine since there will be no filters to display if the filter state is not active. it may be worth considering breaking out the 'save view' button into its own component separate from the `<FiltersChips>` but I wasn't sure if that would fit the scope of this issue